web/docs/tutorials/dev-excuses-app/modifying-main-wasp-file.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Modifying main.wasp file
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+First and foremost, we need to add some dependencies and introduce operations to our project. We’ll add Tailwind to make our UI more pretty and Axios for making API requests. 
+
+Also, we’ll declare a database entity called `Excuse`, queries, and action. The `Excuse` entity consists of the entity’s ID and the text. 
+
+`Queries` are here when we need to fetch/read something, while `actions` are here when we need to change/update data. Both query and action declaration consists of two lines – a reference to the file that contains implementation and a data model to operate on. You can find more info [in the docs section below](language/features#queries-and-actions-aka-operations). So let’s proceed. 
```

Comment:
A lot of concepts introduced at once! I wonder if it is maybe too much at once, possibly hard to follow due to a lot of new words and terms? I am not sure how to improve it though hm. Possibly by introducing concept by concept? Or by explaining them more, but then we go too much into explanations?
Feel free to ignore this comment, since I am not sure there is a much better way.

Reply:
Ok, let's leave it as it is then, maybe we can figure out in the future how to improve it.